logzio-telemetry 3.0.0

ralongit opened the pull request:
- Updated K360 metrics list in `secrets.yaml` - now created dynamically from OOB filters.
- Added `job_dummy` relabel and processtor - Fixing an issue where duplicate metrics were being sent if the metrics were not in the `K8S_360_METRICS` environment variable.
- Use attributes processor to create the `unified_status_code` dimension as it supports connectors.
- **BREAKING CHANGES**:
  - Instead of having the global `metrics.enabled` option of disabling installation of all logzio-telemetry sub charts, each sub chart has its own flag and by default will be installed.

ralongit committed:
Change logzio-telemetry sub chart conditioning (#394)

Instead of having the global `metrics.enabled` option of disabling installation of all logzio-telemetry sub charts, each sub chart has its own flag and by default will be installed.

ralongit committed:
Use attributes processor to create unified_status_code (#395)

Relabel breaks as span metrics is no longer a processor.

ralongit committed:
Fix dup job metrics (#396)

* fix duplicat metrics

* Update charts/logzio-telemetry/README.md

Co-authored-by: Miri Bar <37478344+mirii1994@users.noreply.github.com>

* Update _helpers.tpl

fix newline in secret

---------

Co-authored-by: tamir-michaeli <tamir.michaeli@logz.io>
Co-authored-by: Miri Bar <37478344+mirii1994@users.noreply.github.com>

ralongit committed:
Update changelog

mirii1994 commented on charts/logzio-telemetry/README.md:
```diff
@@ -386,8 +386,20 @@ There are two possible approaches to the upgrade you can choose from:
 ## Change log
+* 3.0.0
+  - Updated K360 metrics list in `secrets.yaml` - now created dynamically from OOB filters.
+  - Added `job_dummy` relabel and processtor - Fixing an issue where duplicate metrics were being sent if the metrics were not in the `K8S_360_METRICS` environment variable.
```
```suggestion
  - Added `job_dummy` relabel and processor - Fixing an issue where duplicate metrics were being sent if the metrics were not in the `K8S_360_METRICS` environment variable.
```

mirii1994 commented on charts/logzio-telemetry/README.md:
```diff
@@ -386,8 +386,20 @@ There are two possible approaches to the upgrade you can choose from:
+  - Added `job_dummy` relabel and processtor - Fixing an issue where duplicate metrics were being sent if the metrics were not in the `K8S_360_METRICS` environment variable.
+  - Use attributes processor to create the `unified_status_code` dimension as it supports connectors.
+  - **BREAKING CHANGES**:
+    - Instead of having the global `metrics.enabled` option of disabling installation of all logzio-telemetry sub charts, each sub chart has its own flag and by default will be installed.
```
Please write here the fields names :)

ralongit committed:
Update charts/logzio-telemetry/README.md

Co-authored-by: Miri Bar <37478344+mirii1994@users.noreply.github.com>

ralongit committed:
Update README.md

ralongit committed:
Update README.md

ralongit committed:
unified_status_code dimesnion fix

Use upsert to create/update its value & take http status code value if unified exists already with grpc status code.
